README.md

```diff
@@ -97,6 +97,45 @@ Depending on your use case, it probably makes sense to store a strong reference
 to the `ScannerViewController` in the presenting View Controller (as seen in
 the Example project).
 
+#### Siri Shortcuts
+
+The scanner example project supports Siri Shortcuts since iOS 12*.
+The User own shortcut opens the app and navigates to the Document Scanner.
+You can find the implementation in the example project.
+
+The implementation works as follows:
+Activate Siri in the project and add a NSUserActivityType identifier in info.plist. 
```

Comment:
Could you please write `NSUserActivityType` like this, with a \` before and after the word? As well at `info.plist` and later again when you are writing about the `AppDelegate.swift`. This is helpful for the reader to understand, that the word is a file or a specific type. Thank you! 🙂

Reply:
Fixed. :)

Reply:
Perfect! Thank you for the changes 🎉